build/utils/yarn.go

```diff
@@ -313,7 +323,7 @@ func buildYarn1Root(packageInfo *PackageInfo, packNameToFullName map[string]stri
 	return rootDependency
 }
 
-// splitNameAndVersion splits package name for package version for th following formats ONLY: package-name@version, package-name@npm:version
+// Splits package name for package version for th following formats ONLY: package-name@version, package-name@npm:version
```

Comment:
I recommend using a grammar checker for comments. This comment should be corrected.